src/Tizen.NUI/src/public/WindowSystem/QuickPanelService.cs

```diff
@@ -0,0 +1,285 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Tizen.NUI.WindowSystem.Shell
+{
+    /// <summary>
+    /// Class for the Tizen quickpanel service.
+    /// </summary>
+    /// This class is need to be hidden as inhouse API.
+    /// <since_tizen> 8 </since_tizen>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class QuickPanelService : IDisposable
+    {
+        private TizenShell _tzsh;
+        private IntPtr _tzshQpService;
+        private int _tzshWin;
+        private bool disposed = false;
+        private bool isDisposeQueued = false;
+
+        /// <summary>
+        /// QuickPanel Type.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public enum QuickPanelType
```

Comment:
looks like this is duplicated as QuickPanelClients one. how about extract this enum as one enum class and use it together ?

Reply:
I changed this to Types.
Yes, this enum is duplicated as QuickPanelClient. But two class (QuickPanelClient and QuickPanelService) are separated, so I think using one class's enum to another class looks weird..